aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -611,14 +613,14 @@ class QConvInt8 final : public c10::OperatorKernel {
         "be greater than 0.")
 
     // Allocate output Tensor and a buffer for QNNPACK to use
-    Tensor output = at::_empty_affine_quantized(
```

Comment:
Separate diff?

Reply:
Should have used `ghexport`. Will do that.